V2/Makefile

```diff
@@ -1,8 +1,31 @@
+.PHONY: help
+help:
+		@echo "---------------HELP-----------------"
+		@echo "To install the dependencies type make install"
+		@echo "To test the project type make test"
+		@echo "To run the local database type make start-local"
+		@echo "To run all comands type make ci"
+		@echo "------------------------------------"
+
+.PHONY: install
 install:
-		@echo "Installing Time Tracker"
+		@echo "=========================================Installing dependencies Time Tracker========================================="
 		npm install
-		pip install --upgrade pip
+		# pip install --upgrade pip
```

Comment:
Question: Why do you comment on this line? Don't we want to update?

Reply:
In some laptop pip upgrade dont func, so that was de the reason, in the newest commit is solved.